openshift/e2e-tests-openshift.sh

```diff
@@ -85,84 +86,105 @@ function timeout() {
   return 0
 }
 
+function install_servicemesh(){
+  header "Installing ServiceMesh"
+
+  # Install the ServiceMesh Operator
+  oc apply -f openshift/servicemesh/operator-install.yaml
+
+  # Wait for the istio-operator pod to appear
+  timeout 900 '[[ $(oc get pods -n openshift-operators | grep -c istio-operator) -eq 0 ]]' || return 1
+
+  # Wait until the Operator pod is up and running
+  wait_until_pods_running openshift-operators || return 1
+
+  # Deploy ServiceMesh
+  oc new-project $SERVICEMESH_NAMESPACE
+  oc apply -n $SERVICEMESH_NAMESPACE -f openshift/servicemesh/controlplane-install.yaml
+  cat <<EOF | oc apply -f -
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: ${SERVICEMESH_NAMESPACE}
+spec:
+  members:
+  - serving-tests
+  - serving-tests-alt
+  - ${SERVING_NAMESPACE}
+EOF
```

Comment:
Can we add ServiceMeshMemberRoll yaml info also in openshift/servicemesh/controlplane-install.yaml ?

Reply:
I refrained from doing that to keep it locally available for the script to change.